### ISSUE TITLE
Add tooltip on hover

### ISSUE THREAD

**Issue by VaninaDzhuteva** [wakirin/Litepicker#89]
Hello,

I love your datepicker! It`s awesome, it has everything what I needed so far. I have a question how can I access the <a> tag with the date in it, when I use onDayHover, so I can add a tooltip on hover? 

Thanks in advance!
Best regards!
Vanina

**Comment by wakirin** [comment]
There is already a tooltip there.
Want to add one more or edit the Litepicker tooltip ?

**Comment by VaninaDzhuteva** [comment]
The default tooltip shows how many nights are choosen when you click on one the dates. We want every single day to have tooltip when you hover over it, that provides another information(how much it costs), so one more tooltip. For now I did it with ::before pseudo element, but when we add languages it will no work anymore.  Can you help with some workaround? Thank you in advance! 
P.S This is the most awesome datepicker project that I found on the web! Cheers!

**Comment by wakirin** [comment]
Perhaps the `onDayHover` event will help here.

https://jsfiddle.net/waki/kpyvatxs/

**Comment by wakirin** [comment]
I will add an event for a tooltip later, since this is interesting case and should be useful.

**Comment by VaninaDzhuteva** [comment]
I just check, that I tried to use onDayHover, but didn`t manage to get the element it self, only the value of it: Mon Jun 29 2020 00:00:00 GMT+0300 (Eastern European Summer Time) for example. And don`t know how to get the anchor tag with class 'day-item'. And the second tooltip will be probably be with different style (bigger, different colors ext.)

**Comment by wakirin** [comment]
use v1.5.4 (just now released)

https://jsfiddle.net/waki/9h6us0L2/
Move mouse on 15, 16, 17, 18 June after select start date.

**Comment by VaninaDzhuteva** [comment]
I am getting an error after I installed the new version: Uncaught TypeError: Cannot read property 'split' of undefined. Is there something that I am missing besides the installation?

**Comment by wakirin** [comment]
Are you sure this error is from Litepicker ?
Your new version is 1.5.4 ?

https://jsfiddle.net/waki/9h6us0L2/
https://wakirin.github.io/Litepicker/

Do you have problem with Litepicker at these links ?

**Comment by VaninaDzhuteva** [comment]
Nope, those links are fine, just in my project. It shows main.js. I am attaching screenshots. Didnt change anything else..
![Capture5](https://user-images.githubusercontent.com/17564311/85021206-344fe880-b17a-11ea-9302-e2fecba62ee3.PNG)
![Capture6](https://user-images.githubusercontent.com/17564311/85021209-34e87f00-b17a-11ea-95b4-04c64c301af3.PNG)
![Capture7](https://user-images.githubusercontent.com/17564311/85021211-35811580-b17a-11ea-8e16-8b78129a5595.PNG)

**Comment by wakirin** [comment]
Post your Litepicker config.

**Comment by VaninaDzhuteva** [comment]
// Init range datepicker
if(document.getElementById('datepicker')) {
	var picker = new Litepicker({ 
		element: document.getElementById('datepicker'),
		numberOfMonths: 3,
		numberOfColumns: 3,
		inlineMode: true,
		mobileFriendly: true,
		selectForward: true,
		singleMode: false,
		showTooltip: true,
		hotelMode: true,
		tooltipText: {
			one: 'night',
			other: 'nights',
		},
		lockDays:  [['2020-05-01', '2020-05-10']], 
		disallowLockDaysInRange: true,
		moveByOneMonth: true,
		onSelect: function(date1, date2) { 
			var dateStart = moment(date1).format('DD/MM/YYYY');
			var dateEnd = moment(date2).format('DD/MM/YYYY');
			document.querySelector('#arrival').value = dateStart;
			document.querySelector('#departure').value = dateEnd;
			document.querySelectorAll('.change-dates-form .form__label').forEach(e => {
				e.classList.add('form__label--filled');
			});
		},

	});
}

**Comment by wakirin** [comment]
https://jsfiddle.net/waki/2huvcp95/

it works, re-check your project.

**Comment by VaninaDzhuteva** [comment]
Hey, I installed several of your past versions. It works fine till 1.4.7 and when I installed 1.5.0 I got this error again. Still don`t know if am I missing something..

**Comment by scott-fisher-iii** [comment]
The latest is v1.5.4 and that is what @wakirin 's latest JSFiddle is using:
https://cdn.jsdelivr.net/npm/litepicker@1.5.4/dist/js/main.js

**Comment by VaninaDzhuteva** [comment]
Ok, but that does not change the fact that until 1.4.7 it work fine in my project. Removed it like npm package and added only with the script tag - got the same result. Try it and in whole new project - still got the same error.

**Comment by gvecchiarelli** [comment]
Hi, I'm getting the same behavior that @VaninaDzhuteva is describing. My project is working with v1.4.7 and it started to throw an error with v1.5.0 or above.

I've detected what causes my project to break in the newer versions, it might be a misconfiguration.

I'm using an inline date picker and I'm setting the calendar's container (a div element) in the `element` property. I can see that the problem goes away if I set the `element` to a button or similar.

It can be reproduced on @wakirin fiddle if you set the `element` to the parent.
```
new Litepicker({
  element: document.getElementById('parent'),
  parentEl: '#parent', // https://github.com/wakirin/Litepicker/issues/100
...
```

What's the appropriate configuration when trying to display the picker in inline mode? What do we need to set to the `element` property if there's no button or input field that we want to rely on?

**Comment by wakirin** [comment]
@glennvdbroeck 
thanks for the additional information.

fixed in v1.5.5
https://jsfiddle.net/waki/9wtmrao2/

**Comment by VaninaDzhuteva** [comment]
Hi, I used onDayHover and add the hover tooltip with diffrent styles and set the price like a data attribute, so I can use it for content in ::before pseudo element. So this is solved for me. Thank you very much!
`
const data = [
  {
    date: '2020-06-15',
    price: '$ 100',
  },
  {
    date: '2020-06-16',
    price: '$ 90',
  },
  {
    date: '2020-06-17',
    price: '$ 15',
  },
  {
    date: '2020-06-18',
    price: '$ 55',
  },
];
var picker = new Litepicker({ 
		element: document.getElementById('datepicker'),
                onDayHover: (date, el, item) => {
			let dateFormatted = moment(date).format('YYYY-MM-DD');
			data.forEach(el => {
				if(dateFormatted === el.date) {
					item.dataset.price = el.price;
				}
			});	
		}
        });`
